action.yml

```diff
@@ -24,6 +24,10 @@ inputs:
     description: 'Fail if the PR is of xl size'
     required: false
     default: 'false'
+  message_if_xl:
+    description: 'Message to show in the PR is xl size'
```

Comment:
```suggestion
    description: 'Message to add as a PR comment in case of surpassing the quantity of changes allowed by the l size, that is, reaching out the xl size'
```